libs/utils/android/workloads/rtapp.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2015, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from android import Wlauto
+from android import Workload
+
+import os
+import glob
+import logging
+
+from shutil import copyfile
+
+class RTApp(Wlauto):
```

Comment:
Overall same comments as per the camerarecord case...

One more consideration, should we better name these workloads to identify that they are provided via WA?
This means:
1. rename the workload to something like: WaRTapp, WaCameRarecord
2. rename the file as well: wa_rtapp.py, wa_camerarecord.py